docs/index.js

```diff
@@ -0,0 +1,621 @@
+      }
+    }
+
+    const P2SHEntry = P2SHEntries[0];
+    const entry = {
+      address: P2SHEntry.address.toString(),
+      amount: kaspaWasm.sompiToKaspaString(P2SHEntry.amount),
+      scriptPublicKey: JSON.parse(P2SHEntry.scriptPublicKey.toString()),
+      blockDaaScore: P2SHEntry.blockDaaScore.toString(),
+      outpoint: JSON.parse(P2SHEntry.outpoint.toString()),
+    };
+
+    const sendPayload = {
+      p: "krc-20",
+      op: "send",
+      tick: document.getElementById("tradeListTick").value.toLowerCase(),
+    };
+
+    const sendScriptBuilder = createKRC20ScriptBuilder(sendPayload);
+    const sendScriptPublicKey = sendScriptBuilder.createPayToScriptHashScript();
+    const sendP2SHAddress = kaspaWasm.addressFromScriptPublicKey(
+      sendScriptPublicKey,
+      "testnet-10",
+    );
+
+    document.getElementById("sendP2SHAddress").innerText =
+      sendP2SHAddress.toString();
+    document.getElementById("sendScript").innerText =
+      sendScriptBuilder.toString();
+
+    const revealTxId = await kastle.signAndBroadcastTx(
+      "testnet-10",
+      [{ amount: "0.3", address: sendP2SHAddress.toString() }],
+      {
+        priorityEntries: [entry],
+        scripts: [
+          {
+            scriptHex: listScriptBuilder.toString(),
+            inputIndex: 0,
+          },
+        ],
+        priorityFee: "1",
+      },
+    );
+
+    document.getElementById("listRevealTxId").innerText = revealTxId;
+    document.getElementById("listErrorKRC20").innerText = "";
+  } catch (error) {
+    document.getElementById("listErrorKRC20").innerText = error.message;
+  } finally {
+    rpc.disconnect();
+  }
+});
+
+document
+  .getElementById("preparedSendTx")
+  .addEventListener("click", async () => {
+    const rpc = new kaspaWasm.RpcClient({
+      url: "wss://ws.tn10.kaspa.forbole.com/borsh",
+      networkId: "testnet-10",
+    });
+    await rpc.connect();
+
+    try {
+      const sendP2SHAddress =
+        document.getElementById("sendP2SHAddress").innerText;
+      const sendScriptBuilder = kaspaWasm.ScriptBuilder.fromScript(
+        document.getElementById("sendScript").innerText,
+      );
+      const sellerAddress = document.getElementById("address").innerText;
+
+      let sendP2SHEntries = [];
+      while (sendP2SHEntries.length === 0) {
+        const sendP2SHUTXOs = await rpc.getUtxosByAddresses([
+          sendP2SHAddress.toString(),
+        ]);
+        sendP2SHEntries = sendP2SHUTXOs.entries;
+      }
+
+      const tx = new kaspaWasm.createTransaction(
+        [sendP2SHEntries[0]],
+        [{ address: sellerAddress, amount: kaspaWasm.kaspaToSompi("1") }],
+        0n,
+      );
+
+      const preparedTxJson = tx.serializeToSafeJSON();
+      const signedTx = await kastle.signTx("testnet-10", preparedTxJson, [
+        {
+          inputIndex: 0,
+          scriptHex: sendScriptBuilder.toString(),
+          signType: "SingleAnyOneCanPay",
+        },
+      ]);
```

Comment:
Using `SingleAnyOneCanPay` sign method allows buyer or market adds new inputs and outputs.